.changeset/wise-feet-play.md

```diff
@@ -1,4 +0,0 @@
----
-'@finos/legend-application-studio': patch
```

Comment:
we don't ever remove changeset, this changeset belongs to another PR right?

Reply:
true 😢